cypress/support/fragments/users/userDefaultObjects/loanDetails.js

```diff
@@ -200,4 +200,18 @@ export default {
       },
     );
   },
+  openFeeFine(amount) {
+    cy.do(Button({ className: including('feefineButton-') }, including(amount)).click());
+  },
+  checkSourceName(row, sourceName) {
+    cy.then(() => MultiColumnListHeader({ id: 'list-column-source' }).index()).then(
```

Comment:
This element is already used in a separate method, it's preferable to store it as a variable